chain/vm/runtime.go

```diff
@@ -490,33 +491,38 @@ func (rt *Runtime) chargeGasFunc(skip int) func(GasCharge) {
 
 }
 
+var enableTracing = false
```

Comment:
Probably shouldn't default to false

(Ideally we'd set this per VM)

Reply:
Yeah, I will work on this in next PR. The performance boost is high enough to justify disabling it until that is done.